mirrord/cli/src/config.rs

```diff
@@ -397,11 +397,20 @@ pub(super) struct PortForwardArgs {
     /// Otherwise, the remote is assumed to be a hostname and lookup is performed in the cluster
     /// after a connection is made to the target.
     #[arg(short = 'L', long)]
-    pub port_mappings: Vec<PortMapping>,
+    pub port_mappings: Option<Vec<AddrPortMapping>>,
```

Comment:
The original reason that mapping args are not positional and use `-L` is to imitate the ssh port forward command as closely as possible, with `-R` then being chosen for the reverse. After some internal discussion, it has been decided that the user should be able to use `-L` and `-R` arguments in the same command. In this light, I can remove the -r flag (you're right, it isn't needed) and go from there - I'll play around with the clap `required`-adjacent stuff to see if it can enforce the presence of at least one of `-L, `-R`